tests/app_builder/with_block.rs

```diff
@@ -0,0 +1,18 @@
+use cosmwasm_std::{BlockInfo, Timestamp};
+use cw_multi_test::AppBuilder;
+
+#[test]
+fn building_app_with_custom_block_should_work() {
+    let mut initialized = false;
+    let app_builder = AppBuilder::default();
+    let _ = app_builder
+        .with_block(BlockInfo {
+            height: 20,
+            time: Timestamp::from_nanos(1_571_797_419_879_305_544),
+            chain_id: "my-testnet".to_string(),
+        })
+        .build(|_, _, _| {
+            initialized = true;
```

Comment:
Keeper tests are looking great now! I think this one should also assert the block info.